diff --git a/lib/coreBundle.js b/lib/coreBundle.js
index d9041dd0f009072511a7d808dec83121ff29e127..889cba8d84bce84e456a3ee4b63e20f17ee75a77 100644
--- a/lib/coreBundle.js
+++ b/lib/coreBundle.js
@@ -39218,7 +39218,7 @@ var init_transport = __esm({
     "use strict";
     init_happyEyeballs();
     init_task();
-    ws = require("./utilsBundle").ws;
+    ws = require("ws");
     perMessageDeflate2 = {
       clientNoContextTakeover: true,
       zlibDeflateOptions: {
